app/src/main/java/fraktal/io/android/demo/timer/ui/TimerView.kt

```diff
@@ -113,7 +116,7 @@ private fun PreviewButtons() {
         Box(modifier = Modifier.padding(it))
         Render(
             timerState = TimerViewState(
-                "02:54",
```

Comment:
![image](https://github.com/jQwout/fmodel-android-demo/assets/37448471/b9efdff9-cf1e-49a6-9131-32d952bc65a1)
its need for preview rendering. This is how it looks in the studio.
ill define random time for checking presentation.  Let's leave it at that

Reply:
uups, sorry